### PR TITLE
Finished grub2_disable_interactive_boot and grub_legacy_disable_interactive_boot rules

### PR DESCRIPTION
**Pull request by matusmarhefka**
#### Description:

Rule grub2_disable_interactive_boot:
* bash and ansible fixes moved under linux_os/guide
* fixed bad ansible remediation
* added test

Rule grub_legacy_disable_interactive_boot:
* bash fix moved under linux_os/guide
* OVAL check moved under linux_os/guide

#### Rationale:

This rule is a part of Fedora OSPP profile.

### CODE REVIEW

File: linux_os/guide/system/accounts/accounts-physical/grub2_disable_interactive_boot/ansible/shared.yml [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+- name: Verify that Interactive Boot is Disabled in /etc/default/grub
+  replace:
+    dest: /etc/default/grub
+    regexp: systemd.confirm_spawn=(1|yes|true|on)
+    replace: systemd.confirm_spawn=no
+  tags:
+    @ANSIBLE_TAGS@
+
+- name: Verify that Interactive Boot is Disabled (runtime)
+  shell: /sbin/grubby --update-kernel=ALL --remove-args="systemd.confirm_spawn"
```
Comment (redhatrises) [inline linux_os/guide/system/accounts/accounts-physical/grub2_disable_interactive_boot/ansible/shared.yml]: @matusmarhefka can you confirm if grubby modifies `/etc/default/grub`?

Comment (dahaic) [reply linux_os/guide/system/accounts/accounts-physical/grub2_disable_interactive_boot/ansible/shared.yml]: @redhatrises I believe it does not care about that file at all. https://bugzilla.redhat.com/show_bug.cgi?id=768106#c6

Comment (redhatrises) [reply linux_os/guide/system/accounts/accounts-physical/grub2_disable_interactive_boot/ansible/shared.yml]: Thanks @dahaic

**Comment by redhatrises** [comment]
ack

**Pull request merged**